src/codegen/expression.rs

```diff
@@ -1421,10 +1438,22 @@ fn revert(
 
     if opt.log_runtime_errors {
         if expr.is_some() {
-            let error_string = error_msg_with_loc(ns, " revert encountered", Some(loc));
+            let prefix = "runtime_error: ".as_bytes();
+            let error_string = format!(
+                " revert encountered in {},\n",
+                ns.loc_to_string(false, &loc)
+            );
             let print_expr = Expression::FormatString(
                 Loc::Codegen,
                 vec![
+                    (
+                        FormatArg::StringLiteral,
+                        Expression::BytesLiteral(
+                            Loc::Codegen,
+                            Type::Bytes(prefix.len() as u8),
+                            prefix.to_vec(),
```

Comment:
Here you should create a new string which is `prefix` and `error_string` concatenated, and then create a single `FormatArg::StringLiteral` rather than two.

Reply:
Please ignore, sorry